lib/iob/calculate.js

```diff
@@ -73,7 +73,7 @@ function iobCalcBiLinear(treatment, minsAgo, dia) {
         iobContrib = treatment.insulin * ( (-0.001852*x1*x1) + (0.001852*x1) + 1.000000 );
 
     } else if (scaled_minsAgo < end) {
-        activityContrib = treatment.insulin * (2 / dia / 60 - (scaled_minsAgo - peak) * 2 / dia / 60 / (60 * 3 - peak));
+        activityContrib = treatment.insulin * (2 / dia / 60 - (scaled_minsAgo - peak) * 2 / dia / 60 / (end - peak));
```

Comment:
Why this change?

Reply:
It seems risky to hard code values in a formula, and less easy to understand what the logic is. If `end` were to be changed, this calculation for `activityContrib` would no longer be correct.

In 071bc9f, I set `activityPeak = 2 / dia / 60` and 
`slopeDown = activityPeak / (end - peak)`.

I think  those changes are helpful because then `2 / dia / 60` isn't duplicated in the code several times, and the overall formula is easier to understand.